widget/embedded/src/components/ConfirmWalletsModal/ConfirmWalletsModal.tsx

```diff
@@ -298,43 +327,43 @@ export function ConfirmWalletsModal(props: PropTypes) {
-            );
+          <Wallets>
+            {requiredWallets.map((requiredWallet, index) => {
+              const blockchain = blockchains.find(
+                (blockchain) => blockchain.name === requiredWallet
+              );
 
-            const key = `wallet-${index}`;
-            return (
-              <div key={key}>
-                <Title>
-                  <Typography
-                    variant="title"
-                    size="xmedium">{`Your ${blockchain?.displayName} wallets`}</Typography>
-                  <Typography
-                    variant="label"
-                    color="$neutral900"
-                    size="medium">{`You need to connect a ${blockchain?.displayName} wallet.`}</Typography>
-                </Title>
-                <Divider size={24} />
-                <ListContainer>
-                  <WalletList
-                    chain={requiredWallet}
-                    isSelected={isSelected}
-                    selectWallet={onChange}
-                    multiWallets={config?.multiWallets ?? true}
-                    supportedWallets={config?.wallets ?? []}
-                    config={config}
-                    limit={NUMBER_OF_WALLETS_TO_DISPLAY}
-                    onShowMore={setShowMoreWalletFor.bind(
-                      null,
-                      blockchain?.name ?? ''
-                    )}
-                  />
-                </ListContainer>
-                {index !== requiredWallets.length - 1 && <Divider size={32} />}
-                {index === requiredWallets.length - 1 &&
-                  customDestinationEnabled && (
-                    <>
+              const key = `wallet-${index}`;
+              return (
+                <div key={key}>
+                  <Title>
+                    <Typography
+                      variant="title"
+                      size="xmedium">{`Your ${blockchain?.displayName} wallets`}</Typography>
+                    <Typography
+                      variant="label"
+                      color="$neutral900"
+                      size="medium">{`You need to connect a ${blockchain?.displayName} wallet.`}</Typography>
+                  </Title>
+                  <Divider size={24} />
+                  <ListContainer>
+                    <WalletList
+                      chain={requiredWallet}
+                      isSelected={isSelected}
+                      selectWallet={onChange}
+                      multiWallets={config?.multiWallets ?? true}
+                      supportedWallets={config?.wallets ?? []}
+                      config={config}
+                      limit={NUMBER_OF_WALLETS_TO_DISPLAY}
+                      onShowMore={setShowMoreWalletFor.bind(
+                        null,
+                        blockchain?.name ?? ''
+                      )}
+                    />
+                  </ListContainer>
+                  {!isLastWallet(index) && <Divider size={32} />}
```

Comment:
There is no need to make it a function, you could move it to line 335 (inside the map for `requiredWallets`)